R/env.R

```diff
@@ -574,29 +676,30 @@ env_get <- function(env = caller_env(), nm, inherit = FALSE) {
 #' vector.
 #' All names are returned, even those starting with a dot.
 #'
-#' @inheritParams env
+#' @inheritParams get_env
 #' @return A character vector of object names.
 #' @export
 #' @examples
-#' env_names()
+#' env <- env(a = 1, b = 2)
+#' env_names(env)
 #' @useDynLib rlang rlang_unescape_character
-env_names <- function(env = caller_env()) {
-  env_ <- get_env(env)
-  x <- names(env_)
-  .Call(rlang_unescape_character, x)
+env_names <- function(env) {
```

Comment:
This function returns the symbols as strings. The name of a symbol is a string in the terminology I've used in rlang. I don't know how to call those otherwise. I'll add some explanation about strings versus symbols, and why environments bind values to symbols rather than strings. Might also be a good place to talk about encoding issues.